<launch>
  <param name="wheel_span"  value="0.9906"/> <!-- Length in meters -->
  <param name="wheel_radius" value="0.14605"/> <!-- Radius in meters -->

  <node pkg="tfr_control" type="drivebase" name="drivebase"/>
</launch>
